client/components/pill/index.js

```diff
@@ -1,10 +1,9 @@
 /**
  * External dependencies
  */
-import React from 'react';
 import styled from '@emotion/styled';
 
-const PillWrapper = styled.span`
```

Comment:
No need for a wrapper, exporting the component directly. This ensures that the tree is smaller.